docs/product-release-notes/src/main/asciidoc/rn-whats-new-con.adoc

```diff
@@ -130,6 +135,17 @@ Initial integration with NoSQL has been introduced. This is based on additional
 === Quartz improvements
 The Quartz scheduler service has been improved to fetch jobs only for deployments (KIE containers) that are currently available in the running system. 
 
+=== SLA due date tracking
+You can now track your processes, the activities in your processes, and cases by service level agreement (SLA) due date.
```

Comment:
Should the SLA phrase have initial caps?

Reply:
No. On cap spelled-out acronyms if they are proper nouns. @michelehaglund